### PR TITLE
Asm 2960 terraform fix gcp and roteted secret

### PR DESCRIPTION
**Pull request by tuvia-akeyless**
not including the test file.

### CODE REVIEW

File: akeyless/resource_gateway_rotated_secret.go [hunk 1]
```diff
@@ -302,74 +289,59 @@ func resourceRotatedSecretRead(d *schema.ResourceData, m interface{}) error {
...
+		} else if isLdapPayload(val) {
+			err = d.Set("user_attribute", fmt.Sprintf("%v", val["ldap_user_attr"]))
+			if err != nil {
+				return err
+			}
+			err = d.Set("user_dn", fmt.Sprintf("%v", val["ldap_user_dn"]))
+			if err != nil {
+				return err
+			}
+		} else if isTargetValue(val) {
```
Comment (renanaAkeyless) [inline akeyless/resource_gateway_rotated_secret.go:322]: remove isTargetValue --> empty condition

Comment (tuvia-akeyless) [reply akeyless/resource_gateway_rotated_secret.go:322]: Fine.
Though, try to consider a future case that we would have to add the condition body, but won't find the correct case. the empty case may save us time of search.

File: akeyless/resource_gateway_rotated_secret.go [hunk 2]
```diff
@@ -53,13 +53,13 @@ func resourceRotatedSecret() *schema.Resource {
...
+				Type:        schema.TypeBool,
 				Required:    false,
 				Optional:    true,
 				Description: "Whether to automatically rotate every --rotation-interval days, or disable existing automatic rotation",
 			},
 			"rotation_interval": {
-				Type:        schema.TypeString,
+				Type:        schema.TypeInt,
 				Required:    false,
 				Optional:    true,
```
[the remark this answers is not in the document]
Comment (tuvia-akeyless) [reply akeyless/resource_gateway_rotated_secret.go:64]: V

File: akeyless/resource_gateway_producer_custom.go [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+	}
+	if rOut.AdminRotationIntervalDays != nil {
+		err = d.Set("admin_rotation_interval_days", *rOut.AdminRotationIntervalDays)
+		if err != nil {
+			return err
+		}
+	}
+
+	/*
+	   // TODO fix this
```
Comment (renanaAkeyless) [inline akeyless/resource_gateway_producer_custom.go:222]: // TODO fix this

Comment (tuvia-akeyless) [reply akeyless/resource_gateway_producer_custom.go:222]: V

**Review by renanaAkeyless — APPROVED** [review]

**Pull request merged**